app/src/main/res/values-ja/strings.xml

```diff
@@ -199,11 +215,13 @@
 
     <string name="pref_status_text_size">トゥートのテキストサイズ</string>
 
-    <!--<string-array name="status_text_size_names">-->
-        <!--<item>小</item>-->
-        <!--<item>中</item>-->
-        <!--<item>大</item>-->
-    <!--</string-array>-->
+    <string-array name="status_text_size_names">-->
```

Comment:
Sorry, I missed this ```-->```...

Reply:
ah thx, I fixed it: https://github.com/tuskyapp/Tusky/commit/197badb13b8d10de789f7a60891a4bd80a9a38c1

Reply:
Thank you!